orbit3d/main.py

```diff
@@ -69,18 +70,19 @@ def initialize_data(config):
 
     data = orbit.Data(HipID, RVFile, AstrometryFile)
     if use_epoch_astrometry:
+        to_jd = lambda x: Time(x, format='decimalyear').jd +.5
```

Comment:
Note that the + half day addition here is only to be consistent with old versions of HTOF which automatically did this. It is not actually true that astropy.time.Time has a half day conversion error from decimal year to julian day. 

We should probably remove this + half day thing.

Reply:
This has been resolved. The half day correction has been removed.